internal/app/ui/cpwsarea/wsmap/pmap/psettings/screenshot.go

```diff
@@ -20,11 +20,13 @@ import (
 	"github.com/SpaiR/imgui-go"
 	"github.com/rs/zerolog/log"
 	"github.com/sqweek/dialog"
+	"golang.design/x/clipboard"
 )
 
 type sessionScreenshot struct {
 	saving          bool
 	inSelectionMode bool
+	toClipboardMode bool
```

Comment:
Actually, I think this could be part of the configuration data. I believe saving to the clipboard is a common use case, so we can set it as a constant preference.

Reply:
i dont think i know how to do that

Reply:
Ok, I'll take a look at it later.